src/interfaces/github.ts

```diff
@@ -224,10 +226,17 @@ interface GitHubInstallation {
 	node_id: number;
 }
 
+//refer from here https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push
+//TODO: tbh, I think we should use a better typing, something more official
+interface GitHubCommit {
+	id: string;
+	message: string;
+}
+
 export interface GitHubPushData {
 	webhookId: string;
 	webhookReceived: number;
 	repository: GitHubRepository;
-	commits: string[];
+	commits: GitHubCommit[];
```

Comment:
How did this work before when it was `string[]`? Shouldn't the code at https://github.com/atlassian/github-for-jira/blob/main/src/github/push.ts#L17 change as a result of this?

Reply:
The code is doing bellow
![image](https://user-images.githubusercontent.com/105693507/183794134-55e54417-dfe8-45e0-969a-9dcd58cd87df.png)

At runtime the code is working regardless, at compile time typing is only `any`.